src/config.rs

```diff
@@ -117,6 +117,28 @@ impl Config {
         }
     }
 
+    pub fn confusable_target(&self, target: &Target) {
+        if std::env::var("CROSS_ALLOW_CONFUSABLE_TARGET").is_ok() {
```

Comment:
When would this be needed?

Reply:
Never I'd hope, but just in case someone actually wants to do this very strange thing and don't want to have output about it, they can disable it.

Realistically this will never be needed.

Reply:
> Realistically this will never be needed.

I'd say let's keep it simple and remove this variable.